cinnabar/util.py

```diff
@@ -608,6 +609,20 @@ def run(func):
         reporter = MemoryReporter()
     try:
         retcode = func(sys.argv[1:])
+    except Exception as e:
+        # Catch all exceptions and provide a nice message
+        retcode = 1
+        sys.stderr.write(
+            'Sorry. The cinnabar extension exited because of an error.\n')
```

Comment:
"The cinnabar extension" works when you do something like `git pull`, but when the command you run is `git cinnabar $command`, that's weird. I think we can skip this message entirely.

Reply:
So - maybe "ERROR: {}".format(e.message) ?